supervisor/backups/manager.py

```diff
@@ -74,12 +79,16 @@ def get(self, slug: str) -> Backup:
 
     def _get_base_path(self, location: Mount | type[DEFAULT] | None = DEFAULT) -> Path:
         """Get base path for backup using location or default location."""
+        if location == DEFAULT and self.sys_mounts.default_backup_mount:
+            location = self.sys_mounts.default_backup_mount
+
         if location:
+            if not location.local_where.is_mount():
+                raise BackupMountDownError(
+                    f"{location.name} is down, cannot back-up to it", _LOGGER.error
+                )
```

Comment:
I guess we could debate over if we should fallback to the backup path here. The Supervisor should have raised a repair issue for the mount at this point. And reading issues, I think most people prefer that the backup fails in this case. Otherwise chances are high that we fill up the local disk.

Reply:
I don't think we should do that, at least not silently. If there were no errors, people will expect the backup to be in the location specified. If we want to do a fallback setup I think we'd need to either one of two things:

1. Tell the user via repair or notification of some kind that we made the backup but it is in the wrong place and they need to correct it once their mount is back online
2. Automatically move the backup for them to the right place once the backup is online

I also think this behavior would need to be configurable. Since users with SD cards would likely prefer backups fail in this situation rather then having us write the whole thing to their data disk.

We can do some of this if we want. But its a new feature and should be a separate PR. This PR is mostly about making the existing feature work the way we intended (writes should fail when the mount is down)

Reply:
Yeah sounds good to me. I think most important is that the user has clear indication that something is off (in this case the mount did not work). The consequences he most likely will realize on his own. I am fine with this :+1: